util/cmd/release/main.go

```diff
@@ -109,9 +109,98 @@ func main() {
 	util.Execute(
 		"ghr",
 		"-t="+token,
+		"-n="+version,
+		"-b='"+notes(version)+"'",
 		"-u=googleapis",
 		"-r=gapic-generator-go",
 		"-c="+commitish,
 		version,
 		stagingDir)
 }
+
+// notes collects the commit messages since the previous tag, aggregates
+// dependency updates into one message, and returns formatted release notes.
+func notes(version string) string {
+	previous := util.ExecuteWithOutput(
+		"git",
+		"describe",
+		"--abbrev=0",
+		"--tags",
+		version+"^",
+	)
+	previous = strings.TrimSpace(previous)
+
+	output := util.ExecuteWithOutput(
+		"git",
+		"log",
+		previous+"..HEAD",
+		"--oneline",
+		"--pretty=format:%s",
```

Comment:
Same comment.

Reply:
There doesn't seem to be a `--no-pager` flag for the `log` cmd ([docs](https://git-scm.com/docs/git-log)). But in my local testing, there hasn't been a problem for the script with longer output.